CONTRIBUTING.md

```diff
@@ -15,10 +15,10 @@ contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additio
 ### Additional Helpful Links for Contributors  
 Many people all over the world have helped make this project better.  You'll want to check out:
 
-* [What are some good first issues for new contributors to the repo?](https://github.com/azure/azure-sdk-for-c/issues?q=is%3Aopen+is%3Aissue+label%3A%22up+for+grabs%22)
-* [How to build and test your change][azure_sdk_for_c_contributing_developer_guide]
-* [How you can make a change happen!][azure_sdk_for_c_contributing_pull_requests]
-* Frequently Asked Questions (FAQ) and Conceptual Topics in the detailed [Azure SDK for Embedded C wiki](https://github.com/azure/azure-sdk-for-c/wiki).
+* [What are some good first issues for new contributors to the repo?](https://github.com/azure/azure-sdk-for-c/issues?q=is%3Aopen+is%3Aissue+label%3Aup-for-grabs)
```

Comment:
The up-for-grabs label has hyphens in it instead of spaces.

Interestingly, looks like GitHub URL encodes white space as `+` within query parameters. cc @JeffreyRichter, @antkmsft

Reply:
Pluses are fine for this.